### PR TITLE
Fixing issue 1598 probably

### PR DESCRIPTION
**Pull request by Sublimis**
* Narrowed synchronizations in RenderTheme (#1598 )

### CODE REVIEW

**Comment by devemux86** [comment]
I merged the pull request in branch [issue_1599](https://github.com/mapsforge/mapsforge/tree/issue_1599) for testing.

@mg4gh Can you also test it?

File: mapsforge-map/src/main/java/org/mapsforge/map/layer/renderer/MapWorkerPool.java [hunk 1]
```diff
@@ -28,19 +28,15 @@
...
 
 public class MapWorkerPool implements Runnable {
     private static final Logger LOGGER = Logger.getLogger(MapWorkerPool.class.getName());
 
-    public static boolean DEBUG_TIMING = false;
-
-    private final AtomicInteger concurrentJobs = new AtomicInteger();
-    private final AtomicLong totalExecutions = new AtomicLong();
-    private final AtomicLong totalTime = new AtomicLong();
+    public static boolean DEBUG_TIMING = true;
```
Comment (devemux86) [inline mapsforge-map/src/main/java/org/mapsforge/map/layer/renderer/MapWorkerPool.java:38]: @Sublimis This should be `false`.

**Comment by Sublimis** [comment]
@devemux86 AFAIC this code is ready for master if you agree

**Comment by devemux86** [comment]
@Sublimis Thanks!

**Pull request merged**